operator_ui/src/containers/Header.tsx

```diff
@@ -135,42 +136,38 @@ const navStyles = ({ palette, spacing }: Theme) =>
     },
   })
 
-const isNavActive = (current?: string, to?: string) =>
-  `${to && to.toLowerCase()}` === current
-
 interface NavProps extends WithStyles<typeof navStyles> {
   authenticated: boolean
-  url?: string
 }
 
-const Nav = withStyles(navStyles)(
-  ({ authenticated, url, classes }: NavProps) => {
-    return (
-      <Typography variant="body1" component="div">
-        <List className={classes.horizontalNav}>
-          {SHARED_NAV_ITEMS.map(([to, text]) => (
-            <ListItem key={to} className={classes.horizontalNavItem}>
-              <BaseLink
-                href={to}
-                className={classNames(
-                  classes.horizontalNavLink,
-                  isNavActive(to, url) && classes.activeNavLink,
-                )}
-              >
-                {text}
-              </BaseLink>
-            </ListItem>
-          ))}
-          {authenticated && (
-            <ListItem className={classes.horizontalNavItem}>
-              <AvatarMenu />
-            </ListItem>
-          )}
-        </List>
-      </Typography>
-    )
-  },
-)
+const Nav = withStyles(navStyles)(({ authenticated, classes }: NavProps) => {
+  const { pathname } = useLocation()
+
+  return (
+    <Typography variant="body1" component="div">
```

Comment:
Typography (as a div) feels not right when you only wrap the list component. Is a `box` component a better choice?